src/api.js

```diff
@@ -12,6 +12,7 @@ function parseMessage(rawMessage) {
     actorName: rawMessage.actor_name,
     args: rawMessage.args,
     kwargs: rawMessage.kwargs,
+    progress: rawMessage.progress ? Number.parseFloat(rawMessage.progress) : null,
```

Comment:
Is it a string ? if yes, this has to be done in remoulade when building the state object, you need to make sure progess is a float.